dev-tools/kubernetes/filebeat/Dockerfile.debug

```diff
@@ -4,7 +4,7 @@ ENV PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/go/bin:/usr/local/go/bin
 
 ENV CGO_ENABLED=0
 
-RUN go install github.com/go-delve/delve/cmd/dlv@v1.21.2
+RUN go install github.com/go-delve/delve/cmd/dlv@v1.23.0
```

Comment:
the dlv changes are unrelated to this PR. They are needed in order to make the debugger working with newer version of go